README.md

```diff
@@ -82,21 +82,21 @@ Example using the default output path for the jar:
 #### Options File
 The options file is described in json (example in `specs/options.json`), and has the following possible values:
 
-|Option|Type|Possible Values|
-|---|---|---|
-|validateInfoLicense|boolean|`true`, `false`|
-|validateInfoDescription|boolean|`true`, `false`|
-|validateInfoContact|boolean|`true`, `false`|
-|validateOperationOperationId|boolean|`true`, `false`|
-|validateOperationDescription|boolean|`true`, `false`|
-|validateOperationTag|boolean|`true`, `false`|
-|validateOperationSummary|boolean|`true`, `false`|
-|validateModelPropertiesExample|boolean|`true`, `false`|
-|validateModelNoLocalDef|boolean|`true`, `false`|
-|validateNaming|boolean|`true`, `false`|
-|pathNamingStrategy|string|`CamelCase`, `HyphenCase`, `UnderscoreCase`|
-|parameterNamingStrategy|string|`CamelCase`, `HyphenCase`, `UnderscoreCase`|
-|propertiesNamingStrategy|string|`CamelCase`, `HyphenCase`, `UnderscoreCase`|
+|Option|Type|Possible Values|Description|
+|---|---|---|---|
+|validateInfoLicense|boolean|`true`, `false`|ensures that there is a license section in the info section|
+|validateInfoDescription|boolean|`true`, `false`|ensures that there is a description attribute in the info section|
+|validateInfoContact|boolean|`true`, `false`|ensures that there is a contact section in the info section|
+|validateOperationOperationId|boolean|`true`, `false`|ensures that there is an operation id for each operation|
+|validateOperationDescription|boolean|`true`, `false`|ensures that there is a description for each operation|
+|validateOperationTag|boolean|`true`, `false`|ensures that there is a tag for each operation|
+|validateOperationSummary|boolean|`true`, `false`|ensures that there is a summary for each operation|
+|validateModelPropertiesExample|boolean|`true`, `false`|ensures that the properties of the Schemas have an example value defined|
+|validateModelNoLocalDef|boolean|`true`, `false`| - not used - |
+|validateNaming|boolean|`true`, `false`|ensures the names follow a given strategy|
+|pathNamingStrategy|string|`CamelCase`, `HyphenCase`, `UnderscoreCase`|naming strategy for paths|
```

Comment:
I'm not sure I like the term "naming strategy" that I used there ;)

I think the good word for it would be "naming convention" maybe? I don't remember why I put strategy.

Let me know what you think. Your changes are good but if you agree, I will create a new issue to change the name of the parameter but still support the old one for a while.

Reply:
Ok, good idea.

I have already written "Naming convention" in the description.

Reply:
See https://github.com/OpenAPITools/openapi-style-validator/issues/56 for the follow up…